### ISSUE TITLE
Add three lines to the sample config.json

### ISSUE THREAD

**Issue by daniels303** [NCAR/esp32-atomlite-arduino-iotwx#17]
In order to use the latest version of the code with the RG15 connected to the grove connector, these three lines need to be added to the config.json

    "iotwx_gpio_config": "A",
    "iotwx_use_wifi": "1",
    "iotwx_poe_mac": "0xFFFFFFFFFFFF",

These lines and their description needs to be added to our documentation.

**Comment by kmaull-ucar** [comment]
This is already in the config.json, I think the issue was when you pulled
the latest, that file didn't get updated?

**Comment by daniels303** [comment]
Yes, the issue is that our docs don't have these three lines or the description of how to use them.

See [https://ncar.github.io/openiotwx/installation/core_install/](https://ncar.github.io/openiotwx/installation/core_install/)
